### PR TITLE
Added this file

### PR DESCRIPTION
**Pull request by MrErik0**
@vkudum1 what is  your favorite food?

### CODE REVIEW

**Review by vkudum1 — CHANGES_REQUESTED** [review]
you need to make more changes to get :1st_place_medal:

File: BunnyTest [hunk 1]
```diff
@@ -0,0 +1 @@
+THis is for training
```
Comment (vkudum1) [inline BunnyTest:1]: add new line here<--

Comment (vkudum1) [reply BunnyTest:1]: updated to add a line in split view

Comment (MrErik0) [reply BunnyTest:1]: you are right

**Review by MrErik0 — COMMENTED** [review]
changed

**Review by vkudum1 — APPROVED** [review]
Approving

**Pull request merged**